bundles/remote_services/remote_service_admin_dfi/src/remote_service_admin_dfi.c

```diff
@@ -56,13 +57,30 @@
 #define RSA_LOG_DEBUG(admin, msg, ...) \
     celix_logHelper_log((admin)->loghelper, CELIX_LOG_LEVEL_ERROR, (msg),  ##__VA_ARGS__)
 
+
+/**
+ * If set to true the rsa will create a thread to handle stopping of service export.
+ *
+ * This stop thread can be removed when the branch feature/async_svc_registration is merged and
+ * celix_bundleContext_stopTrackerAsync is available.
+ *
+ */
+#define CELIX_RSA_USE_STOP_EXPORT_THREAD true
```

Comment:
Why a define, and not a property of the bundle?

Reply:
there is no need to change this runtime. For now the stop export thread is needed, but I want to keep the older functionality to see if this can be removed when the async update is merged.